packages/google_sign_in/google_sign_in_ios/example/ios/Podfile

```diff
@@ -28,6 +28,8 @@ require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelpe
 flutter_ios_podfile_setup
 
 target 'Runner' do
+  pod 'GoogleSignIn', :git => 'https://github.com/google/GoogleSignIn-iOS.git', :commit => '0dc783f36577f5bc249056f835d592ffae59d0ee'
```

Comment:
This will not be checked in, it is just for testing and will be removed before this is merged when GoogleSignIn 6.2 is actually released in the next week.  See https://github.com/google/GoogleSignIn-iOS/issues/23#issuecomment-1124253013